### ISSUE TITLE
Dynamic prefixes

### ISSUE THREAD

**Issue by jakerobinson** [jordansissel/fpm#846]
I'd like to supply a dynamic prefix to fpm so that when I build a gem, it uses the install dir of GEM_HOME on the client installing the RPM.

Given a machine with RVM using ruby-2.2
When I install a gem packaged with fpm
Then the gem will be located in the GEM_HOME of RVM ruby-2.2

**Comment by jordansissel** [comment]
Agreed. Using `-s gem` should set the prefix attribute so that when building an rpm it is relocatable. You can then install using the rpm/yum prefix (relocation?) feature

**Comment by jakerobinson** [comment]
The only way I can think to do this is a prefix of /tmp/gemname, then a post-install script to file the folder to $GEM_HOME and $GEM_HOME/bin. I'll give it a shot and post my results.
